### PR TITLE
[AP-2207] slack notif: point users to the appropriate files to edit

### PR DESCRIPTION
**Pull request by chouquette**
### What does this PR do?

Add some minor clarifications on how to fix `slack_teams_channels_check` jobs failures

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [x] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature. 
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Comment by Pythyu** [comment]
👋 Nice PR !
Since you changed an error message only used by Datadog, you can add the label `qa/skip-qa` otherwise it will generate QA cards at the next release :) And there's nothing to QA there 
And You'll need to add a milestone as well usually the next available agent version, for us it would be 7.48.0 since we are in code freeze right now.
I'll let you fix that but gonna approve anyway, you'll be able to merge once the merge freeze is lifted next week

**Review by Pythyu — APPROVED** [review]

**Comment by chouquette** [comment]
Thanks a lot for the clarifications! I updated the PR accordingly :)

**Pull request merged**
